low-level/sspl-ll.spec

```diff
@@ -140,6 +141,13 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
 [ -f $SSPL_DIR/sspl_init ] ||
     ln -s $SSPL_DIR/bin/sspl_provisioner_init $SSPL_DIR/sspl_init
 
+# Creating softlink uinder site-packages to use sspl module easier way.
+[ -d "${SSPL_DIR}" ] && {
+    mkdir -p /usr/lib/python3.6/site-packages/cortx/sspl
+    ln -s /opt/seagate/cortx/sspl/bin /usr/lib/python3.6/site-packages/cortx/sspl/bin
```

Comment:
This is a duplicate code. Same code is being checked in the post_install PR as well. 
https://github.com/Seagate/cortx-monitor/pull/279